src/BDSMagnetStrength.cc

```diff
@@ -164,7 +178,22 @@ const std::map<G4String, BDSMagnetStrength::unitsFactors> BDSMagnetStrength::uni
     {"rmat41"        , {"",    1.0}},
     {"rmat42"        , {"",    1.0}},
     {"rmat43"        , {"",    1.0}},
-    {"rmat44"        , {"",    1.0}}
+    {"rmat44"        , {"",    1.0}},
+    {"cavity_efield" , {"",    1.0}},
+    {"cavity_travelling", {"",    1.0}},
+    {"cavity_radius"    , {"",    CLHEP::m}},
+    {"cavity_length"    , {"",    CLHEP::m}},
+    {"cavity_m"      , {"",    1.0}},
+    {"cavity_n"     , {"",    1.0}},
+    {"cavity_p"     , {"",    1.0}},
```

Comment:
presumably these should have some units here?

Reply:
These do not have any units, they are mode numbers